examples/ut/two_scenario/controller.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef CONTROLLER_HPP
+#define CONTROLLER_HPP
+
+#include <forsyde.hpp>
+
+typedef std::function <std::vector<int>(const std::vector<int>&)> scenrio_func;
+
+///!< gamma function for the zipU process of the top module
+unsigned int gamma_func_zipa (const unsigned int &ca)
```

Comment:
Change all the size values, such as the return values of gamma functions, to `size_t`.